destinations/destinations.go

```diff
@@ -0,0 +1,40 @@
+package destinations
+
+import (
+	"io"
+	"net"
+	"os"
+)
+
+var (
+	connection io.WriteCloser
+)
+
+func NewRemote() (io.WriteCloser, error) {
+	destination := readLoggingDestination()
+
+	return getConnection(destination)
+}
+
+func readLoggingDestination() string {
+	destination := os.Getenv("FISKALY_REMOTE_LOGGING_DESTINATION")
+	if destination == "" {
+		panic("env var FISKALY_REMOTE_LOGGING_DESTINATION not defined.")
+	}
+	return destination
+}
+
+func getConnection(destination string) (io.WriteCloser, error) {
+	if connection == nil {
```

Comment:
where is connection set, other than with getConnection? do we intend to call NewRemote multiple times? I would suggets separating the connection constructor from the simple getter, or, in this case, simply rename getConnection to 'establishConnection' or similar, since I don't see a use case where the simple getter is needed (unless I am missing smth)

Reply:
it is just set in in `getConnection`. `NewRemote` seems to be called multiple times from mice, since the function that calls it is part of the `newHandler` function.
I have renamed the `getConnection` to `establishConnection`.

Reply:
The idea with this was to make the connection a lazy-loaded singleton. The problem being that if it wasn't one had to somehow ensure that the connection will get closed. Mice creates a new logger instance for each request which in turn then would also create a new connection per request which would quickly fill up the available ports on the host machine.